docs/init-config.sample.yaml

```diff
@@ -41,72 +41,71 @@ LCoreAlloc:
-  # Ethernet device MTU
+  # Ethernet device MTU.
   EthMtu: 1500
-  # Ethernet RX queue capacity
+  # RX queue capacity for Ethernet faces.
   EthRxqFrames: 4096
-  # Ethernet before-TX queue capacity
+  # Before-TX queue capacity for Ethernet faces.
   EthTxqPkts: 256
-  # Ethernet after-TX queue capacity
+  # After-TX queue capacity for Ethernet faces.
   EthTxqFrames: 4096
 
-  # whether to enable socket faces
+  # Whether to enable socket faces.
   EnableSock: true
-  # socket before-TX queue capacity
+  # Before-TX queue capacity for socket faces.
   SockTxqPkts: 256
-  # socket after-TX queue capacity
+  # After-TX queue capacity for socket faces.
   SockTxqFrames: 1024
 
-  # ChanRxGroup queue capacity (shared among all socket/mock faces)
+  # ChanRxGroup queue capacity, shared among all socket/mock faces.
   ChanRxgFrames: 4096
 
-# NDT
+# NAME DISPATCH TABLE
 Ndt:
-  # Names are dispatched using prefix with this number of components.
+  # Names are dispatched using a prefix with this number of components.
   PrefixLen: 2
-  # There are (2^IndexBits) elements.
+  # There are 2^IndexBits elements in the table.
   IndexBits: 16
-  # Collect per-element sample every (2^SampleFreq) lookups. Must not exceed 30.
+  # Collect per-element sample every 2^SampleFreq lookups. Must not exceed 30.
   SampleFreq: 8
 
-# FIB
+# FORWARDING INFORMATION BASE
 Fib:
-  # Capacity in each partition. Affects hugepage memory usage.
-  # (2^q-1) is optimal.
+  # Capacity in each partition. Affects hugepage memory usage. 2^q-1 is optimal.
   MaxEntries: 65535
-  # Number of hash table buckets in each partition. Must be power of 2.
+  # Number of hash table buckets in each partition. Must be a power of 2.
   NBuckets: 256
-  # 'M' parameter in 2-stage LPM algorithm.
-  # Should be greater than length of most FIB entry names.
+  # 'M' parameter in the 2-stage LPM algorithm.
+  # Should be greater than the length of most FIB entry names.
   StartDepth: 8
 
 # FORWARDER DATA PLANE
 Fwdp:
-  # Settings of Interest queue between FwInput and FwFwd.
+  # Settings for the Interest queue between FwInput and FwFwd.
   FwdInterestQueue:
     Capacity: 131072
     DequeueBurstSize: 32
-  # Settings of Data queue between FwInput and FwFwd.
+  # Settings for the Data queue between FwInput and FwFwd.
   FwdDataQueue:
     Capacity: 131072
     DequeueBurstSize: 64
-  # Settings of Nack queue between FwInput and FwFwd.
+  # Settings for the Nack queue between FwInput and FwFwd.
   FwdNackQueue:
     Capacity: 131072
     DequeueBurstSize: 64
   # Collect RX-FwFwd latency sample every 2^LatencySampleFreq packets.
   # Must not exceed 30.
   LatencySampleFreq: 16
-  # PIT suppression config.
+  # PIT suppression settings.
   Suppress:
     Min: 10ms
     Max: 100ms
     Multiplier: 2.0
-  # Capacity of PCCT. (2^q-1) is optimal.
+  # Capacity of the PCCT. 2^q-1 is optimal.
   PcctCapacity: 131071
   # Capacity of CS in-memory direct entries. Twice as many PCCT entries could
```

Comment:
Unclear what "capacity of ... entries" means. If I remember correctly this is the (max) number of entries, can you confirm/clarify?

Reply:
"capacity" and "max number of entries" are the same concept.

Reply:
Yes they are. But the current text says "capacity of entries" which sounds like "max size of _each_ entry" and is quite different from "max _number_ of entries"

Reply:
All these CsCap* settings are collectively "CS capacity". There are four types of CS entries, and each type has a separate capacity limit.

First letter:

* M = in-memory
* D = on-disk

Second letter:

* D = direct entry
* I = indirect entry

Reply:
unless there's a very strong reason for this unfriendly "encoding" of the config keys, I suggest a future refactoring of this portion as follows:

```yaml
CsCapacity:
  MemDirect: ...
  MemIndirect: ...
  DiskDirect: ...
  DiskIndirect: ...
```

Reply:
Well, don't bother with `init-config.yaml` anymore. It will be deleted in favor of JSON-RPC "activation" commands.
I'll consider this suggestion when designing the equivalent command.

Reply:
sure